application-ckeditor-plugins/src/main/resources/xwiki-dialog/modal.js

```diff
@@ -99,3 +99,31 @@ define('modal', ['jquery', 'l10n!modal', 'bootstrap'], function($, translations)
     createModalStep: createModalStep
   };
 });
+
+/**
+ * Utility module to load required skin extensions.
+ */
+define('xwiki-skinx', ['jquery'], function($) {
+  'use strict';
+
+  $.fn.loadRequiredSkinExtensions = function(requiredSkinExtensions) {
```

Comment:
Deserves a comment explaining that it's not needed since https://jira.xwiki.org/browse/XWIKI-18895 and should be removed when upgrading the parent version.